sdks/java/io/kafka/src/test/java/org/apache/beam/sdk/io/kafka/KafkaIOTest.java

```diff
@@ -529,7 +529,7 @@ public void testUnboundedSourceCustomTimestamps() {
                   (tp, prevWatermark) -> new CustomTimestampPolicyWithLimitedDelay<Integer, Long>(
                     (record -> new Instant(TimeUnit.SECONDS.toMillis(record.getKV().getValue())
                                              + customTimestampStartMillis)),
-                   Duration.millis(0),
+                   Duration.ZERO,
```

Comment:
Just curious: ErrorProne suggested this?

Reply:
Yes.
It offers some nice suggestions for readability I've found (e.g. seconds(180) -> minutes(3))

Reply:
Very nice!